src/sources/host_metrics/network.rs

```diff
@@ -15,11 +15,22 @@ use super::{filter_result, FilterList, HostMetrics};
 #[configurable_component]
 #[derive(Clone, Debug, Default)]
 pub struct NetworkConfig {
-    /// Lists of device name patterns to include or exclude.
+    /// Lists of device name patterns to include or exclude in gathering
+    /// network utilization metrics.
+    ///
+    /// Defaults to including all devices.
     #[serde(default)]
+    #[configurable(metadata(docs::examples = "example_devices()"))]
     devices: FilterList,
 }
 
+fn example_devices() -> FilterList {
+    FilterList {
+        includes: Some(vec!["sda".try_into().unwrap()]),
+        excludes: Some(vec!["dm-*".try_into().unwrap()]),
+    }
+}
```

Comment:
Seems like this is a duplicate of the one in `disk.rs` . Perhaps they could be extracted to mod.rs ?